packages/protocol/src/peer/Peer.ts

```diff
@@ -84,13 +86,14 @@ export class Peer {
      * Permanently forget the peer.
      */
     async delete() {
-        await this.close();
+        logger.info("Removing", Diagnostic.strong(this.toString()));
+        await this.close(false);
         await this.#context.deletePeer(this);
         await this.#context.sessions.deleteResumptionRecord(this.address);
     }
 
     /**
-     *
+     * Remove the peer
```

Comment:
is the comment really correct? because basically only sessions for the peer are removed, or?

Reply:
It looks like the comment is unfinished but this does more than remove sessions; it stops discovery and reconnection, terminates sessions and removes it from the PeerSet